NetCoreRestApi/DataLayer/Repositories/IProductRepository.cs

```diff
@@ -0,0 +1,8 @@
+﻿using DataLayer.Models;
+
+namespace DataLayer.Repositories
+{
+    public interface IProductRepository : IRepository<ProductModel>
```

Comment:
Create interfaces folder and put all interfaces there

Reply:
done